<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>6</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>4</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0" colspan="2">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>应用程序设置</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>应用名称</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditAppName">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>界面风格</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEditStyle">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>数据库设置</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="2">
         <widget class="QPushButton" name="pushButtonInitDatabase">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>初始化数据库</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1" colspan="2">
         <widget class="QLineEdit" name="lineEditDBName">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>118</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="6" column="1" colspan="2">
         <widget class="QLineEdit" name="lineEditDBUserID">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>数据库名称</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>数据库账号</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1" colspan="2">
         <widget class="QLineEdit" name="lineEditDBUserPassword">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>30</height>
           </size>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>服务器地址</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" colspan="2">
         <widget class="QLineEdit" name="lineEditIP">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>服务器端口</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1" colspan="2">
         <widget class="QLineEdit" name="lineEditPort">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>数据库密码</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>数据库驱动</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QComboBox" name="comboBoxDBDriver">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>148</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>更新设置</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>4</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>6</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditAppName</tabstop>
  <tabstop>lineEditStyle</tabstop>
  <tabstop>pushButtonInitDatabase</tabstop>
  <tabstop>comboBoxDBDriver</tabstop>
  <tabstop>lineEditIP</tabstop>
  <tabstop>lineEditPort</tabstop>
  <tabstop>lineEditDBName</tabstop>
  <tabstop>lineEditDBUserID</tabstop>
  <tabstop>lineEditDBUserPassword</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
